app/imports/components/ActionItems/CloseOrReopenActionItem/CloseOrReopenActionItem.jsx

```diff
@@ -39,10 +41,12 @@ const CloseOrReopenActionItem = ({
             >
                 <ErrorsField />
                 { isOpen ?
-                    <TextField
+                    <CustomTextField
                         name="closeMessage"
                         floatingLabelText="Close message - optional"
                         fullWidth
+                        value={closeMessage}
```

Comment:
It's working fine. I can save change message also no touched message is saved thanks this prop